src/Components/Navbar.jsx

```diff
@@ -1,21 +1,26 @@
 import { Link } from 'react-router-dom';
-import style from '@/Components/styles/Navbar.module.css'
+import '../Components/styles/Navbar.css'
 
 
 function Navbar() {
   return (
-    <nav className={style["navbar-container"]}>
-      <ul className={style["nav-items"]}>
-      <h1 className={style.heading}>
-        <Link className={style.links} to="/">BookStore CMS</Link>
+    <nav className="navbar-container">
+      <h1 className="BookStore-cms">
+            BookStore CMS
       </h1>
-        <li className={style["link-item"]}>
-          <Link className={style.links} to="/">BOOKS</Link>
-        </li>
-        <li className={style["link-item"]}>
-          <Link className={style.links} to="/categories"> CATEGORIES</Link>
-        </li>
-      </ul>
+        <ul className="nav-items">
+          
+            <li className="link-item">
+              <Link className="links" to="/">BOOKS</Link>
+            </li>
+            <li className="link-item">
+              <Link className="links" to="/categories"> CATEGORIES</Link>
+            </li>
```

Comment:
- [ ] Please make sure you reduce the space between your nav items as shown below
![Uploading 2.png…]()